### PR TITLE
✨ [NO-TICKET] Add SPM support

### PR DESCRIPTION
**Pull request by timmikeljmonzo**
- Adds SPM support 

tested with pointing to the latest commit on this branch & compiles successfully

Screenshot 2022-07-22 at 09 39 58

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    products: [
+        .library(
+            name: "BKMoneyKit",
+            targets: ["BKMoneyKit"]),
+    ],
+    targets: [
+        .target(
+            name: "BKMoneyKit",
+            resources: [
+                .copy("BKMoneyKit.bundle/CardLogo/amex@2x.png"),
```
Comment (timmikeljmonzo) [inline Package.swift]: I wonder if just making this `process("BKMoneyKit.bundle/CardLogo")` would auto process all resources in that folder 🤔

Comment (monchote) [inline Package.swift]: I think for images is better to use `.process`. 

From the [docs](https://developer.apple.com/documentation/xcode/bundling-resources-with-a-swift-package):
```
Process rule
For most use cases, use [process(_:localization:)](https://developer.apple.com/documentation/PackageDescription/Resource/process(_:localization:)) to apply this rule and have Xcode process the resource according to the platform you’re building the package for. For example, Xcode may optimize image files for a platform that supports such optimizations. If you apply the process rule to a directory’s path, Xcode applies the rule recursively to the directory’s contents. If no special processing is available for a resource, Xcode copies the resource to the resource bundle’s top-level directory.

Copy rule
Some Swift packages may require a resource file to remain untouched or to retain a certain directory structure for resources. Use the [copy(_:)](https://developer.apple.com/documentation/PackageDescription/Resource/copy(_:)) function to apply this rule and have Xcode copy the resource as is to the top level of the resource bundle. If you pass a directory path to the copy rule, Xcode retains the directory’s structure.
```

**Review by monchote — CHANGES_REQUESTED** [review]
Looks great! Just a minor thing

File: Package.swift (hunk 1 above)
Comment (monchote) [reply Package.swift]: I think that would work. You can also use `.process` with individual items.

Comment (timmikeljmonzo) [reply Package.swift]: I will just point to the `BKMoneyKit.bundle` as everything in there are just images

**Review by monchote — APPROVED** [review]
Built it locally and all seems good.

Screenshot 2022-07-22 at 10 52 38

File: Package.swift (hunk 1 above)
Comment (timmikeljmonzo) [reply Package.swift]: Just tested an it compiles OK with just pointing to `BKMoneyKit.bundle` ✅

**Pull request merged**